### PR TITLE
Add warning to 2.0 kubeadm to use 2.1

### LINKED ISSUE #648: Warn Calico-v2.0 users to use Calico v2.1 with kubeadm-1.6
We have a separate manifest for k8s 1.6 for Calico v2.1:
http://docs.projectcalico.org/v2.1/getting-started/kubernetes/installation/hosted/kubeadm/

but not for Calico v2.0

(one of the users just tripped on that)

### PR DESCRIPTION
**Pull request by tmjd**
Fixes #648

### CODE REVIEW

File: v2.0/getting-started/kubernetes/installation/hosted/kubeadm/index.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ own etcd.
 
 You can easily create a cluster compatible with this manifest by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
 
+> **Note:**
+>   For the best Calico experience, when using kubeadm 1.6+ use at least
```
Comment (gunjan5) [inline v2.0/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: `use at least` could be reworded as `use [Calico *v*2.1] or higher`

File: v2.0/getting-started/kubernetes/installation/hosted/kubeadm/index.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ own etcd.
...
+[Calico 2.1]({{site.baseurl}}/v2.1/getting-started/kubernetes/installation/hosted/kubeadm/).
```
Comment (gunjan5) [inline v2.0/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: should be `v2.1`

**Comment by tmjd** [comment]
@gunjan5 PTAL

**Comment by gunjan5** [comment]
LGTM

**Pull request merged**